src/utils/AuthService.ts

```diff
@@ -124,7 +123,12 @@ export async function initializeAuthenticationProd(
     validateUser: UserValidationFunc,
     isSigninCallback: boolean
 ) {
-    const idpSettings = await idpSettingsFetcher();
+    const idpSettings = await idpSettingsFetcher().then((r) =>
+        // backward compatibility with old callers
+        Object.prototype.toString.call(r) !== '[object Response]'
```

Comment:
Erf... I would rather avoid that kind of glue... Can we just go back to previous behavior ?

Reply:
okay